### PR TITLE
add list part

### PR DESCRIPTION
**Pull request by maxNia**
code working fine!
video in telegram

### CODE REVIEW

**Review by CatDidenko — CHANGES_REQUESTED** [review]
Please, divide relations to the separate files. Contacts relations move to contact file, List relations move to the list model.

File: src/listOfContacts/listOfContacts.model.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { DataTypes, Model } from 'sequelize';
+import sequelize from "../config/database";
```
Comment (CatDidenko) [inline src/listOfContacts/listOfContacts.model.ts]: single quotes

File: contactService/config/server.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader('Access-Control-Allow-Headers', 'origin, content-type, accept');
+  res.setHeader('Access-Control-Allow-Methods', '*');
+  next();
+});
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+
```
Comment (CatDidenko) [inline contactService/config/server.ts]: Remove extra empty lines

File: contactService/contact/contact.controller.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  } catch(e: any) {
+    next(e);
+  }
+}
+
+export async function getContacts(req: Request, res: Response, next: NextFunction) {
+  try {
+    const contacts = await contactService.getContacts();
+    res.status(200).send(contacts)
+  } catch(e: any) {
```
Comment (CatDidenko) [inline contactService/contact/contact.controller.ts]: remove any

File: contactService/email/mail.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import { Router } from "express";
+import nodemailer from 'nodemailer';
+
+const routers = Router();
+
+routers.get('/mail', mailToAll);
+
+async function mailToAll(req: any, res: any, next: any) {
+  console.log('hello');
```
Comment (CatDidenko) [inline contactService/email/mail.ts]: remove console.log

Comment (CatDidenko) [reply contactService/email/mail.ts]: Actually, this functionality should be in the separate microservice (separate express project)

Comment (CatDidenko) [reply contactService/email/mail.ts]: Please, remove this file

File: contactService/email/mail.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import { Router } from "express";
```
Comment (CatDidenko) [inline contactService/email/mail.ts]: single quotes, please

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: src/listOfContacts/listOfContacts.model.ts (hunk 1 above)
Comment (maxNia) [reply src/listOfContacts/listOfContacts.model.ts]: updated

File: contactService/config/server.ts (hunk 2 above)
Comment (maxNia) [reply contactService/config/server.ts]: updated

File: contactService/email/mail.ts (hunk 5 above)
Comment (maxNia) [reply contactService/email/mail.ts]: file deleted

File: contactService/email/mail.ts (hunk 4 above)
Comment (maxNia) [reply contactService/email/mail.ts]: deleted

File: contactService/contact/contact.controller.ts (hunk 3 above)
Comment (maxNia) [reply contactService/contact/contact.controller.ts]: removed

**Pull request merged**